### ISSUE TITLE
[Server thread/ERROR] [com.raoulvdberge.refinedstorage.apiimpl.network.NetworkNodeManager/]: Could not read network node

### ISSUE THREAD

**Issue by seals55** [raoulvdberge/refinedstorage#2511]
#### Issue description:
Server crashed trying to remove stuck build in Build Monitor. Now I can't start my server.
#### What happens:
When server almost finishes loading, preparing world 0% it sits there for a little while then has a huge java.lang.StackOverflowError: null error
#### What you expected to happen:
The server to start normally
#### Steps to reproduce:
1. Stick pattern in exporter with autocraft upgrade.
2. Eventually pattern gets stuck in build queue, Build Monitor doesn't report an issue it just never finishes
3. Server crashes when clearing build monitor
4. Server won't start up again
...

#### Version (make sure you are on the latest version before reporting):

- Minecraft: 1.15.2
- Forge: 31.1.74
- Refined Storage: 1.8.3

Does this issue occur on a server? [yes/no] 
yes

#### If a (crash)log is relevant for this issue, link it here:

[debug.log](https://gist.github.com/seals55/1c7fde8b08ee781b8a03eaf97e6ef28a)

**Comment by seals55** [comment]
The first log is actually not the server crashing, just pausing until the watchdog kills it. I increased the watchdog tick time. 
Restarted and the server came up in about 5 minutes. Still had the same Could not read network node error, and sat there for a few minutes...but eventually it worked. The server seemed fine when I joined.

Requested a build and the build hung. I had the build monitor UI open...then the server crashed (I didn't even click to cancel).

[crash-2020-05-06_23.35.04-server.txt](https://github.com/raoulvdberge/refinedstorage/files/4590596/crash-2020-05-06_23.35.04-server.txt)
[debug.log](https://github.com/raoulvdberge/refinedstorage/files/4590599/debug.log)
